composer.json

```diff
@@ -17,8 +17,8 @@
         "guzzlehttp/guzzle": "^6.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.7",
-        "arbor-education/php-coding-standard": "^1.0",
+        "phpunit/phpunit": "^4.7 || ^9.0",
```

Comment:
Since this has a minimum version of PHP7, we should modify the PHPUnit 4.x dependency as its upper bound of support is PHP 5.6, and PHPUnit 9 requires PHP 8.

To verify this is working install PHP 7.1, then update the dependencies using composer in a similar fashion to this: `php7.1 $(which composer) update --prefer-lowest`

Reply:
How did this work before then? We didn't change the `^4.7` requirement, we just added `^9.0` for PHP 8 support, which shouldn't affect anything else. How does going from `^4.7` to `^4.7 || ^9.0` change the lower bound?

Reply:
Just add `"phpunit/phpunit": ">=4.7"` in order to support any version based on PHP version.
`"phpunit/phpunit": "^4.7 || ^9.0"` makes no sense as it doesn't allow anything between and there is no reason for that.

Reply:
Added exactly that, can you all guys please review?

@igorvuckovic @AnthonyPorthouse @davorzdralo @gecbla